build/Build.Pack.cs

```diff
@@ -477,12 +477,13 @@ void PackTarballs(string runtimeId)
             $"tentacle-{FullSemVer}-{NetCore}-{runtimeId}.tar.gz");
     }
 
-    void BuildAndPushOrLoadKubernetesTentacleContainerImage(bool push, bool load)
+    void BuildAndPushOrLoadKubernetesTentacleContainerImage(bool push, bool load, string? host = null)
     {
+        host = host is not null ? $"{host}/" : null;
         DockerTasks.DockerBuildxBuild(settings =>
             settings.AddBuildArg($"BUILD_NUMBER={FullSemVer}", $"BUILD_DATE={DateTime.UtcNow:O}")
                 .SetPlatform("linux/arm64,linux/amd64")
-                .SetTag($"docker.packages.octopushq.com/octopusdeploy/kubernetes-tentacle:{FullSemVer}")
+                .SetTag($"{host}octopusdeploy/kubernetes-tentacle:{FullSemVer}")
```

Comment:
Is there a missing slash (or something) here?

Reply:
I sub in the slash on line 482 because if it's null we don't want the leading slash 👍